Deduplication --> Antispam (#293)
transparency-dev/static-ct: deployment/live/aws/test/README.md

Terraforming the account can be done by:
  1. `cd` to [/deployment/live/aws/test/](/deployment/live/aws/test/) to deploy/change.
  2. Run `terragrunt apply`. If this fails to create the antispam database,

>   connect the RDS instance to your VM using the instrunctions bellow, and run

Reviewer: ```suggestion
  connect the RDS instance to your VM using the instructions below, and run
```

  --db_port=3306 \
  --db_user=tesseract \
  --db_password=${TESSERACT_DB_PASSWORD} \

>   --antispam_db_name=antispam_db

Reviewer: ```suggestion
  --antispam_db_name=antispam_db \
```

  --db_port=3306 \
  --db_user=tesseract \
  --db_password=${TESSERACT_DB_PASSWORD} \

>   --antispam_db_name=antispam_db

Reviewer: ```suggestion
  --antispam_db_name=antispam_db \
```
